RELEASE_NOTES.md

```diff
@@ -1,5 +1,36 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
+# Chef Client Release Notes 14.2:
+
+## `ssh-agent` support for user keys
+
+You can now use `ssh-agent` to hold your user key when using knife. This allows storing your user key in an encrypted form as well as using `ssh -A` agent forwarding for running knife commands from remote devices.
+
+You can enable this by add `ssh_agent_signing true` to your `knife.rb` or `ssh_agent_signing = true` in your `credentials` file.
```

Comment:
`by adding`.

Reply:
You know you wrote this right?

Reply:
I did mention that :) In my defense I think I wrote that whole feature in like 24 hours straight through.